FirebaseAuth/Sources/Swift/AuthProvider/PhoneAuthProvider.swift

```diff
@@ -236,7 +236,7 @@ import Foundation
                                                  retryOnInvalidAppCredential: Bool,
                                                  multiFactorSession session: MultiFactorSession?,
                                                  uiDelegate: AuthUIDelegate?) async throws
-      -> String? {
+      -> String {
```

Comment:
This return type bubbles up to the public API, which we need to make non-optional if possible, so this is why this is being changed to non-optional String.